app/build.gradle

```diff
@@ -26,4 +26,5 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    compile project(path: ':smslibrary')
```

Comment:
Depending on the gradle version compile might be deprecated and replaced with implementation

Reply:
Thanks, I let AndroidStudio take care of this automatically and it out `compile` instead of `implementation`. Will definetly change.